ramls/profile_picture.json

```diff
@@ -13,6 +13,11 @@
       "description": "byte array of image",
       "type": "string",
       "format": "base64"
+    },
```

Comment:
Do we need hmac property here ?

Reply:
Yes , we need . As we need to store some checksum for profile picture Object to maintain its integrity.